### ISSUE TITLE
SyntaxError: Unexpected token import

### ISSUE THREAD

**Issue by jay-jlm** [cezarpretto/cnpj-promise#1]
I get this error when trying to run a Node project which uses this lib as a dependency:

`
/Users/joao/node_projects/my-project/node_modules/cnpj-promise/src/cnpj-promise.js:2
import Promise from 'promise'
^^^^^^
SyntaxError: Unexpected token import
    at Object.exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:543:28)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/joao/node_projects/kapmug-proto-server/node_modules/babel-watch/runner.js:63:7)
`

The solution is to configure a babel npm script and run babel inside of the module's directory to generate a transpiled version to a `dist` folder.

Is there a reason this is not provided by default?

**Comment by cezarpretto** [comment]
Hello. 
Like this is a recent project, I don't had implemented this yet. But, soon as possible, I think until the weekend I will do that.

**Comment by cezarpretto** [comment]
@jay-at-avido I just published the version 3.0.0 and it's working on node. You can use this way:

```js
var cnpjPromise = require('cnpj-promise').default

cnpjPromise('47508411022559').then(data => {
  console.log(data);
}).catch(err => {
  console.error(err);
})
```

It's only update the package to 3.0.0
